src/UserEmail.php

```diff
@@ -220,12 +220,12 @@ public function showChildForItemForm($canedit, $field_name, $id, bool $display =
         if ($this->fields['is_default']) {
             $result .= " checked";
         }
-        $result .= ">&nbsp;";
+        $result .= " aria-label='" . __s('Set as default email') . "'>&nbsp;";
```

Comment:
It seems it would be wrong when user cannot edit.

Reply:
I think it is OK as long as the `aria-disabled='true'` is present when the user cannot edit.